src/lzma_worker.js

```diff
@@ -2628,7 +2628,8 @@ var LZMA = (function () {
     return {
         /** xs */
         compress:   compress,
-        decompress: decompress
+        decompress: decompress,
+        terminateWorker: function() {}
```

Comment:
I assume this was added to match lzma.js, but I'm wondering if maybe leaving it out is better since calling this terminateWorker() doesn't do anything. Perhaps it's better to just let it throw an error and let the coder check to see if it exists. Thoughts?